evap/results/tools.py

```diff
@@ -160,11 +160,17 @@ def avg_distribution(distributions, weights=itertools.repeat(1)):
 
 
 def average_grade_questions_distribution(results):
-    return avg_distribution([normalized_distribution(result.counts) for result in results if result.question.is_grade_question])
+    return avg_distribution(
+        [normalized_distribution(result.counts) for result in results if result.question.is_grade_question],
+        [sum(result.counts) for result in results if result.question.is_grade_question]
```

Comment:
there is `result.total_count`, which is admittedly weird since it does barely more than `return sum(counts)`, but we need it for the templates and i would suggest using it everywhere.

...maybe `sum_of_counts` would be a better name, if you think so as well please rename it.

Reply:
i now called it `count_sum` because we already have `*_sum` at other places